backend/endpoints/userAuth.js

```diff
@@ -67,10 +67,21 @@ app.post("/create-user", async (req, res) => {
 
 /***
  * Attempts to log in using the provided credentials.
- * Returns 200 for successful log in attempt and a 401 otherwise.
+ * Returns 200 for successful log in attempt and a 500 otherwise.
```

Comment:
So it's not necessarily that we have to use a 500 status code if login fails. 500 is like something is wrong with the server. For a failing login, that's typically a 401. You can read more on status codes here: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#server_error_responses

Reply:
Got it